docs/static/advanced-pipeline.asciidoc

```diff
@@ -117,15 +118,8 @@ filter {
     }
 }
 --------------------------------------------------------------------------------
-And change output to see a result :
-[source,json]
---------------------------------------------------------------------------------
-output {
-  stdout { codec => rubydebug }
-}
---------------------------------------------------------------------------------
```

Comment:
@guyboertje I do think that the rubydebug codec is useful, but the tutorial works without it. I removed it from the tutorial because it's not part of the final config file that we tell users to build. My guess is that someone meant to remove it entirely but missed a spot. Users don't know what a codec is yet, so introducing it without more explanation is potentially confusing.  I'm planning more improvements to the tutorial in 5.0, including a more iterative approach to building the parts of the config file so that users can see the results of their work without having to build everything first. When I make those changes, it might make sense to add rubydebug back into the mix. The way it was written before, though, it was confusing.